website/marietje/models.py

```diff
@@ -0,0 +1,180 @@
+import os
+
+from django.contrib.auth import get_user_model
+from django.db import models
+from django.conf import settings
+from spotipy import SpotifyOAuth
+from spotipy.client import Spotify
+
+User = get_user_model()
+
+
+class SpotifySettings(models.Model):
```

Comment:
I think `SpotifySettings` is a bit unclear. Maybe call it `SpotifyAccounts` (because that's basically what it is). You add/link Spotify accounts to django and then you can use those at certain venues.